# ROS-standard
GPSFix:          {topic: gps,            frame_id: gps}
NavSatFix:       {topic: fix,            frame_id: gps} 
IMU:             {topic: imu/data,       frame_id: imu_link}
RAWIMU:          {topic: imu/data_raw,   frame_id: imu_link}
Odometry:        {topic: odom,           frame_id: odom} 
OdometryOrigin:  {topic: odom_origin,    frame_id: odom} 

# Oem7-specific 
OemRawMsg:         {topic: oemraw,          frame_id: gps, queue_size: 200}
BESTGNSSPOS:        {topic: bestgnsspos,      frame_id: gps}
BESTPOS:            {topic: bestpos,          frame_id: gps}
BESTUTM:            {topic: bestutm,          frame_id: gps}
BESTVEL:            {topic: bestvel,          frame_id: gps}
PPPPOS:             {topic: ppppos,           frame_id: gps}
TERRASTARSTATUS:    {topic: terrastarstatus,  frame_id: gps}
TERRASTARINFO:      {topic: terrastarinfo,    frame_id: gps}
CORRIMU:            {topic: corrimu,          frame_id: gps}
CORRIMUDATA:        {topic: corrimudata,      frame_id: gps}
HEADING2:           {topic: heading2,         frame_id: gps}
INSPVA:             {topic: inspva,           frame_id: gps}
INSPVAX:            {topic: inspvax,          frame_id: gps,  queue_size: 10}
INSSTDEV:           {topic: insstdev,         frame_id: gps,  queue_size: 10}
INSCONFIG:          {topic: insconfig,        frame_id: gps,  queue_size: 10}
RXSTATUS:           {topic: rxstatus,         frame_id: gps,  queue_size: 10}
TIME:               {topic: time,             frame_id: gps}
BYINS2:             {topic: byins2,           frame_id: gps,  queue_size: 10}
